contribs.md

```diff
@@ -1,6 +1,6 @@
 # Contributing Guidelines
 
-To contribute to either the documentation or stubs, you can fork & clone our repository, make your edits, and push back to GitHub and send us a pull request. All items that get pulled into the main repository will automatically get pushed to [our.umbraco.com/documentation](https://our.umbraco.com/documentation).
+To contribute to either the documentation or stubs, you can fork & clone our repository, make your edits, and push back to GitHub and send us a pull request. All items that get pulled into the main repository will automatically get pushed to [docs.umbraco.com/umbraco-cms](https://docs.umbraco.com/umbraco-cms/).
```

Comment:
As per our [style guideline](https://docs.umbraco.com/welcome/contribute/style-guide#long-sentences), we need to write shorter sentences with less than 25 words so am just making a small change suggestion here with addition that it should link to the main docs page instead of cms docs (as contributions are for all umbraco products, not just cms)
```suggestion
To contribute to either the documentation or stubs, you can fork & clone our repository. Then you make your edits, and push back to GitHub and send us a pull request. All items that get pulled into the main repository will automatically get pushed to [docs.umbraco.com](https://docs.umbraco.com).
```